backend/operations_scanner.go

```diff
@@ -151,6 +159,16 @@ func NewOperationsScanner(dbClient database.DBClient, ocmConnection *ocmsdk.Conn
 		s.lastOperationTimestamp.WithLabelValues(v)
 	}
 
+	for _, v := range []arm.SubscriptionState{
+		arm.SubscriptionStateRegistered,
+		arm.SubscriptionStateUnregistered,
+		arm.SubscriptionStateWarned,
+		arm.SubscriptionStateDeleted,
+		arm.SubscriptionStateSuspended,
```

Comment:
I found it handy to have a "generator" function (in Python lingo) for ARM resource provisioning states - [ListProvisioningStates](https://github.com/Azure/ARO-HCP/blob/a16e7926fb42e87d22bcffc595f04156794dd0c0/internal/api/arm/resource.go#L130-L142) - to avoid having to write out all the constants.

I could see a similar "ListSubscriptionStates" function being handy too.  It would save you from having to write out all the constants here and in `collectSubscriptions`.